src/tests/JIT/Regression/JitBlue/Runtime_117605/Runtime_117605.cs

```diff
@@ -0,0 +1,45 @@
+// Generated by Fuzzlyn v3.3 on 2025-07-14 11:26:52
+// Run on X64 Windows
+// Seed: 5654087083843205658-vectort,vector128,vector256,x86aes,x86avx,x86avx2,x86avx512bw,x86avx512bwvl,x86avx512cd,x86avx512cdvl,x86avx512dq,x86avx512dqvl,x86avx512f,x86avx512fvl,x86avx512fx64,x86bmi1,x86bmi1x64,x86bmi2,x86bmi2x64,x86fma,x86lzcnt,x86lzcntx64,x86pclmulqdq,x86popcnt,x86popcntx64,x86sse,x86ssex64,x86sse2,x86sse2x64,x86sse3,x86sse41,x86sse41x64,x86sse42,x86sse42x64,x86ssse3,x86x86base
+// Reduced from 33.4 KiB to 0.9 KiB in 00:01:30
+// Hits JIT assert for Release:
+// Assertion failed '(maskBaseSize == 4) || (maskBaseSize == 8)' in 'Program:M0()' during 'Rationalize IR' (IL size 87; hash 0xaf50ff37; FullOpts)
+//
+//     File: D:\a\_work\1\s\src\coreclr\jit\gentree.cpp Line: 20819
+//
+using System.Numerics;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+public class C0
+{
+    public short M3(ref Vector<uint>[] arg0, Vector256<ulong> arg1)
+    {
+        var vr5 = Vector128.Create(0, -1, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 0, 0, 0, 0);
+        return (short)Sse2.MoveMask(vr5);
+    }
+}
+
+public class Program
```

Comment:
Shouldn't the test use xunit? e.g. [Fact].
Also, we typically name the main class to be e.g. Runtime_117605.
Also, the standard .NET Foundation license header

Reply:
Yes and yes. I thought I had done this, but clearly I haven't 😅